webserver/src/main/java/io/kestra/webserver/controllers/TriggerController.java

```diff
@@ -0,0 +1,44 @@
+package io.kestra.webserver.controllers;
+
+import io.kestra.core.models.triggers.Trigger;
+import io.kestra.core.repositories.TriggerRepositoryInterface;
+import io.kestra.webserver.responses.PagedResults;
+import io.kestra.webserver.utils.PageableUtils;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.QueryValue;
+import io.micronaut.http.exceptions.HttpStatusException;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.scheduling.annotation.ExecuteOn;
+import io.micronaut.validation.Validated;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import jakarta.inject.Inject;
+
+import java.util.List;
+
+@Validated
+@Controller("/api/v1/triggers")
+public class TriggerController {
+    @Inject
+    private TriggerRepositoryInterface triggerRepository;
+
+    @ExecuteOn(TaskExecutors.IO)
+    @Get(uri = "/search", produces = MediaType.TEXT_JSON)
+    @Operation(tags = {"Triggers"}, summary = "Search for triggers")
+    public PagedResults<Trigger> search(
+        @Parameter(description = "The current page") @QueryValue(defaultValue = "1") int page,
+        @Parameter(description = "The current page size") @QueryValue(defaultValue = "10") int size,
+        @Parameter(description = "The sort of current page") @Nullable @QueryValue List<String> sort,
+        @Parameter(description = "A string filter") @Nullable @QueryValue(value = "q") String query,
+        @Parameter(description = "A namespace filter prefix") @Nullable @QueryValue String namespace
+    ) throws HttpStatusException {
+        return PagedResults.of(triggerRepository.find(
+            PageableUtils.from(page, size, sort, triggerRepository.sortMapping()),
```

Comment:
As sortMapping is the identity function you can use the overload without a sortMapper

Reply:
I don't get it, it actually does things for JDBC part

Reply:
(see AbstractJdbcTriggerRepository)

Reply:
Well, i'm pretty sure that without it it do the same as it's the identity function.